### PR TITLE
v0.2.3 Release

### PR DESCRIPTION
**Pull request by patrickcleeve2**
Version 0.2.3

### CODE REVIEW

File: fibsem/alignment.py
Comment (Rohit-Kannachel) [inline fibsem/alignment.py:1]: dx move for eucentric

Comment (patrickcleeve2) [reply fibsem/alignment.py:1]: ty

Comment (Rohit-Kannachel) [reply fibsem/alignment.py:1]: sorry i meant to say, why is there dx move for eucentric??

**Review by Rohit-Kannachel — APPROVED** [review]
lgtm except not sure about eucentric move

**Review by patrickcleeve2 — COMMENTED** [review]
lgtm

**Pull request merged**